src/aiida_pseudo/cli/install.py

```diff
@@ -297,91 +438,65 @@ def cmd_install_pseudo_dojo(
         'upf': UpfData,
     }
 
-    # yapf: disable
-    paw_configurations = (
-        PseudoDojoConfiguration('1.0', 'PBE', 'SR', 'standard', 'jthxml'),
-        PseudoDojoConfiguration('1.0', 'PBE', 'SR', 'stringent', 'jthxml'),
-        PseudoDojoConfiguration('1.0', 'LDA', 'SR', 'standard', 'jthxml'),
-        PseudoDojoConfiguration('1.0', 'LDA', 'SR', 'stringent', 'jthxml')
-    )
-    # yapf: enable
-
     try:
         pseudo_type = pseudo_type_mapping[pseudo_format]
     except KeyError:
         echo.echo_critical(f'{pseudo_format} is not a valid PseudoDojo pseudopotential format.')
 
     configuration = PseudoDojoConfiguration(version, functional, relativistic, protocol, pseudo_format)
     label = PseudoDojoFamily.format_configuration_label(configuration)
-    description = f'{configuration} installed with aiida-pseudo v{__version__}'
 
     if configuration not in PseudoDojoFamily.valid_configurations:
         echo.echo_critical(f'{configuration} is not a valid configuration')
 
-    if not download_only and QueryBuilder().append(PseudoDojoFamily, filters={'label': label}).first():
-        echo.echo_report(f'{PseudoDojoFamily.__name__}<{label}> is already installed.')
-        sys.exit(1)
-
     with tempfile.TemporaryDirectory() as dirpath:
 
         dirpath = pathlib.Path(dirpath)
+
         filepath_archive = dirpath / 'archive.tgz'
         filepath_metadata = dirpath / 'metadata.tgz'
 
-        download_pseudo_dojo(configuration, filepath_archive, filepath_metadata, traceback)
+        if from_download is not None:
 
-        description += f'\nArchive pseudos md5: {md5_file(filepath_archive)}'
-        description += f'\nPseudo metadata md5: {md5_file(filepath_metadata)}'
+            tarball_path = pathlib.Path(from_download).absolute()
+            with tarfile.open(tarball_path, 'r') as handle:
+                handle.extractall(dirpath)
+
+            filepath_configuration = dirpath / 'configuration.json'
+
+            with filepath_configuration.open('r') as handle:
+                configuration = PseudoDojoConfiguration(**json.load(handle))
+                pseudo_type = pseudo_type_mapping[configuration.pseudo_format]
+        else:
```

Comment:
Here you use an `else` but for the SSSP implementation you do another if after the check that the configuration is valid. Maybe make the two follow the same logic?

Reply:
For sure, I agree! I only rebased/updated the SSSP so far and pushed the commit since it was getting late and I wanted to continue working on it at PSI. ^^

So the PR wasn't ready for review yet, sorry about that. Typically I'll (re-)request a review once a PR is fully ready.